src/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+import ReactDOM from "react-dom";
+import marked from "marked";
+import "./index.css";
+
+class MarkdownViewer extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      markdown : ''
+    }
+  }
+  handleChange(event) {
+    this.setState({markdown: marked.parse(event.target.value)})
+  }
+
+  render() {
+    const { markdown } = this.state;
+    return (
+      <div>
+        <textarea rows ="20" cols ="50" className ="data" placeholder="Enter the text" onChange={this.handleChange.bind(this)}></textarea> 
+        <p className ="markdown" dangerouslySetInnerHTML={{__html: markdown}}></p>
```

Comment:
I also did the same thing while doing this. But could you try considering the html you got from marked as jsx? That may help you avoid using dangerouslySetInnerHTML

Reply:
Since this is the first time I am trying I used dangeroulySetInnerHTML. I am not sure of any other alternatives as of now.